### DISCUSSION TITLE
ACCEDE AL GRUPO DE LA MISIÓN FRONTEND EN MICROSOFT TEAMS

### DISCUSSION THREAD

**Discussion by JuanRodiles** [Launch-X-Latam/OnBoarding#4] [Announcements]
# ACCEDE AL GRUPO DE LA MISIÓN FRONTEND EN MICROSOFT TEAMS 

Ahora que has concluido tu OnBoarding, completa el siguiente formulario y te llegará un correo con el acceso a la *Misión FrontEnd*. No olvides usar tus datos de acceso que te proporcionamos en el correo de Bienvenida. 

[Entra a tu grupo en Microsoft Teams](https://forms.office.com/r/Ddr5fxJbNk)

![onboarding](https://innovaccion.cloud/wp-content/uploads/2022/09/Avatar_LX.png)

**Comment by pjdiaz9731** [comment]
Alguien podría ayudarme con una consulta, ya que me indica que no tengo acceso al formulario y no puedo ingresar a la misión por teams

**Reply by GonDev1** [reply]
Buen dia 

En el correo donde mencionan el "On boarding" viene la liga para ingresar al formulario, entras y colocas el link a tu repositorio y listo.

Espero lo puedas resolver pronto  saludos.

**Reply by pjdiaz9731** [reply]
Hola gracias, pero de igual manera sigo sin poder ingresar, me indica que no tengo acceso al formulario y no le ha llegado ningún correo a mi cuenta sobre los datos.

**Reply by pjdiaz9731** [reply]
El punto es que todavía no me han enviado el correo de bienvenida con los datos de acceso.

**Reply by Murillas** [reply]
Lo buscaste en spam? Normalmente el asunto es "atención viajeros"

**Reply by pjdiaz9731** [reply]
Correcto lo busque en spam y solo tengo el correo de cuando iban a estar mandando los datos de acceso luego de ahí no me han enviado nada 🥺

**Reply by pjdiaz9731** [reply]
Ya hice la consulta por correo y me dijeron que el día de ayer 23 lo iba a estar enviando, pero no tengo cómo acceder a nada deste entonces 🥺💔

**Reply by Ezcainer** [reply]
Si aún no puedes o tienes alguna duda , también podrías enviar un correo sobre tu caso 

contacto@innovaccion.mx

Yo envíe un correo por qué tampoco podía unirme. Me ayudaron con el proceso.

**Reply by pjdiaz9731** [reply]
Ya ya hice la consulta por Instagram y correo y nadie me ha respondido la petición

**Reply by BelenneleB** [reply]
Bue creo que tenemos el mismo problema aunque no estoy muy segura, antes que nada la culpa fue completamente mía  pues escribí mal mi correo en el formulario, así que es imposible que reciba más datos para seguir avanzando en el curso espero que no sea así y puedan darme una pronta respuesta para solucionar mi error pues mandé un correo explicando lo que sucedió pero aún no recibo ninguna noticia y es completamente lamentable que por mi estupidez no pueda empezar un curso de algo que realmente me apasiona y me llama el interés.

**Reply by pjdiaz9731** [reply]
Que mal, y si yo tampoco he recibido respuesta, ya he intentado de todas la manera a y nada. 💔

**Reply by BelenneleB** [reply]
Solo queda una opción llorar en silencio escuchando a maná y obvio leer la biblia 😔✌️

**Reply by pjdiaz9731** [reply]
Y es que nadie da atención al correo 💔😞

**Reply by BelenneleB** [reply]
Lo más probable es que están demasiado saturados de correos como para leer los nuestros y es comprensible pues el curso va dirigido a toda la comunidad de Latinoamérica ojalá tengamos buena suerte y todo salga bien

**Reply by BelenneleB** [reply]
> Y es que nadie da atencion al correo💔😞

Confirmo jsjjsjs

**Reply by HJavierMendez** [reply]
si ya llenaron el formulario de entrar al grupo en teams , entren a outlook con el correo de innovaccion y en su bandeja les debio llegar un correo con el el link para agregar el grupo, lo acabo de realizar hoy y entre al grupo en teams espero y les funcione saludos
![Sin título](https://user-images.githubusercontent.com/99088624/192925262-0234f7aa-5770-499d-96ba-e90bfbb3ff14.png)

**Reply by Analauramx** [reply]
para que te manden ese correo debes entrar primero aqui
![aqui](https://user-images.githubusercontent.com/114636568/193117791-14e15e54-cd3e-453c-b10e-9e0b9fc01eb0.png)

**Reply by egoz-98** [reply]
Intenta abriendo la pestaña en incognito

**Comment by GonDev1** [comment]
Mi nombre es Gonzalo y soy de México,  tengo un perro pug que se llama pumba, disfruto mucho del rock alternativo y estoy emocionado por comenzar con esta misión,  exito a todos y seguimos en contacto.

**Comment by LuceroLuciano** [comment]
Hola explorers!, ¿Cómo estan?

¿Ustedes pudieron acceder a la misión frontend en cuanto le dieron click al pase de acceso espacial?
Lo que sucede es que sigo sin equipo, ya envié el formulario y me llego el correo con  el pase de abordar pero aun no formo parte del team, solo me apareció un cuadro de dialogo con el mensaje solicitud enviada, seré paciente, espero que para mañana ya este dentro del equipo, ojala que no haya hecho nada malo. 

Saludos explorers.

**Reply by rowleyx** [reply]
Yo estoy igual, sin equipo, alguien ya sabe cómo?

**Reply by rowleyx** [reply]
![Screenshot_2022-09-30-18-03-16-248_com.microsoft.teams.jpg](https://user-images.githubusercontent.com/113216805/193367532-fb6a81ad-ebcb-4b18-8035-0e76bc17b999.jpg)solo se queda así,

**Comment by leirusi** [comment]
Hola compañeros, a mi tampoco me deja ver el formulario para entrar al Microsoft Teams. ¿Qué han hecho para lograrlo?

**Reply by pjdiaz9731** [reply]
Hola, al final ya logré tener respuesta, hice la consulta por correo y lograron asignarme los datos de acceso, 

Pueden hacer la consulta por le mismo medio espero tengan suerte.

**Reply by JasonAlanya** [reply]
Hola, en mi caso se solucionó **abriendo una pestaña en incognito** e iniciando sesión con la cuenta que nos mandaron desde esta pestaña.

Espero te funcione

**Reply by rowleyx** [reply]
Jaja de que formulario hablan?  ![Screenshot_2022-09-30-18-01-46-633_com.microsoft.teams.jpg](https://user-images.githubusercontent.com/113216805/193367399-ec2a8d51-dd33-465c-96b8-3c88f562e6bc.jpg) solo me sale esto en Teams.

**Comment by Danfel009** [comment]
veo que estamos todos iguales!
